testLibOPHD/testLibOPHD.vcxproj

```diff
@@ -163,6 +163,7 @@
   </ItemDefinitionGroup>
   <ItemGroup>
     <ClCompile Include="main.cpp" />
+    <ClCompile Include="MapOffset.cpp" />

```

Comment:
In the NAS2D unit test project, test files usually have the ending ".test.cpp". Not sure if that standard is super important, though it does make the filename tabs visually distinct when both the main file and the test file are open in an editor.

----

Test projects need at least one source file. That meant including dummy `main.cpp` files for empty projects. Once at least one valid test source file is added, the dummy `main.cpp` can be removed. (Might be best to handle that as a separate followup commit).